### DISCUSSION TITLE
What is the “status”  in the error of phenoage_calc

### DISCUSSION THREAD

**Discussion by Meowmeow656** [dayoonkwon/BioAge#28] [Q&A] [answered]
Hello  and thank you very much for your generous sharing!  When I use the phenoage_calc function to calculate phenotyping age using other datasets, my dataset already has 9 types of biomakers and age, but I always get an error. 

The error message is as follows: "Error in eval(predvars, data, env): object 'status' not found". I really don't understand what 'status' means here. 

Your insights and assistance on this matter would be greatly appreciated.

Below is an example of the code:
> library(haven)
> bio_data <- read_dta("D:/R_bioage.dta")
> phenoage = phenoage_calc(bio_data, biomarkers <- c( "Alb", "Cre", "lnCRP", "AP", "LP", "MSCV", "Glu", "RBC", "WBC"), fit = NULL, orig = FALSE)
Error in eval(predvars, data, env) : object 'status' not found

**Comment by dayoonkwon** [comment, marked as answer]
The error you're encountering, "object 'status' not found", refers to the absence of a "status" variable in your training dataset. This variable is essential for the phenoage calculation because it's based on a Cox regression model, which requires both "time" (time to event, in this case, death) and "status" (aging-related mortality status) to estimate the hazard of aging-related mortality.

To resolve this issue, please ensure your dataset includes both a "time" variable and a "status" variable that represents aging-related mortality status. For a more detailed understanding of how this model is constructed, I recommend reviewing the Levine 2018 paper.
